ios/PurchasesHybridCommon/PurchasesHybridCommon/Purchases+HybridAdditions.swift

```diff
@@ -12,15 +12,15 @@ import RevenueCat
 @objc public extension Purchases {
 
     @objc(configureWithAPIKey:appUserID:purchasesAreCompletedBy:userDefaultsSuiteName:platformFlavor:
-            platformFlavorVersion:usesStoreKit2IfAvailable:dangerousSettings:shouldShowInAppMessagesAutomatically:
+            platformFlavorVersion:storeKitVersion:dangerousSettings:shouldShowInAppMessagesAutomatically:
             verificationMode:)
     static func configure(apiKey: String,
                           appUserID: String?,
                           purchasesAreCompletedBy: PurchasesAreCompletedBy,
                           userDefaultsSuiteName: String?,
                           platformFlavor: String?,
                           platformFlavorVersion: String?,
-                          usesStoreKit2IfAvailable: Bool = false,
+                          storeKitVersion: String? = "DEFAULT",
```

Comment:
This is a great point! When I look at it further, `StoreKitVerison.init(name: String)` can't handle a `nil` value, so I'll modify it to be of type `String` instead of `String?`. Good find!